olm-catalog/serverless-operator/project.yaml

```diff
@@ -14,6 +14,9 @@ olm:
     default: 'stable'
     list:
       - 'stable'
+  previous:
+    replaces: 1.27.0
+    skipRange: '>=1.27.0 <1.28.0'
```

Comment:
I'm assuming that this needs to be bumped whenever project.version is bumped? Should we increase the comment on project.version?

We can follow up on this, no reason to block this PR

Reply:
Yes. It needs to be bumped as you said. Will add the comment in a follow-up.